engine/apps/schedules/models/custom_on_call_shift.py

```diff
@@ -274,6 +276,56 @@ def event_is_finished(self):
 
         return is_finished
 
+    def _daily_by_day_to_ical(self, time_zone, start, users_queue):
+        """Create ical weekly shifts to distribute user groups combining daily + by_day."""
+        result = ""
+        # keep tracking of (users, day) combinations, and starting dates for each
+        combinations = []
+        starting_dates = []
+        # we may need to iterate several times over users until we get a seen combination
+        cycle_users = itertools.cycle(users_queue)
+        orig_start = last_start = start
+        all_rotations_checked = False
+        # we need to go through each individual day
+        day_by_day_rrule = copy.deepcopy(self.event_ical_rules)
+        day_by_day_rrule["interval"] = 1
+        for users in cycle_users:
+            for i in range(self.interval):
+                if not start:  # means that rotation ends before next event starts
+                    all_rotations_checked = True
+                    break
+                last_start = start
+                day = CustomOnCallShift.ICAL_WEEKDAY_MAP[start.weekday()]
+                if (users, day, i) in combinations:
```

Comment:
Could there be any issues if there are identical user groups in `users_queue`? Seems like in this case `(users, day, i)` could be identical and the loop would exit earlier than expected?